src/main/main.ts

```diff
@@ -90,10 +94,30 @@ const createWindow = async () => {
     } else {
       mainWindow.show();
     }
+
+    pyServer = startServer();
+
+    if (pyServer !== null && pyServer.stderr !== null) {
+      pyServer.stderr.once('data', (data) => {
+        console.log(data.toString()); // prints address that the server is running on. TODO: remove before it is electron app is built
+        get('http://localhost:5000', (res) => {
```

Comment:
Can we extract the port into a constant so it's configurable?

Reply:
Of course, its now in the .env file.